.github/workflows/test.yml

```
@@ -0,0 +1,15 @@
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  test:
+    uses: ./.github/workflows/test-sdk.yml
+    with:
+      node-versions: '[18, 19, 20, 21, 22]'
+      upload-coverage: 'true'
```

actionlint (1.7.7)

input "upload-coverage" is typed as string by reusable workflow "./.github/workflows/test-sdk.yml". bool value cannot be assigned

(expression)